colossalai/shardformer/modeling/bert.py

```diff
@@ -109,5 +111,24 @@ def bert_flash_attention_forward(
         if self.is_decoder:
             outputs = outputs + (past_key_value,)
         return outputs
-    
-    return bert_flash_attention_forward
\ No newline at end of file
+
+    return forward
+
+
+def _get_jit_fused_output_forward():
```

Comment:
This closure does not have any local variables. What's the reason to use closure?

Reply:
> This closure does not have any local variables. What's the reason to use closure?

updated.